cmd/up.go

```diff
@@ -360,3 +359,18 @@ func printSvc(w io.Writer, c kubernetes.Interface, nss []string) error {
 	fmt.Fprintln(w)
 	return nil
 }
+
+func allReady(c kubernetes.Interface) (bool, error) {
+	deps, err := c.AppsV1beta1().Deployments(api.NamespaceAll).List(metav1.ListOptions{
+		LabelSelector: "created-by=kubeapps",
+	})
+	if err != nil {
+		return false, err
+	}
+	for _, d := range deps.Items {
```

Comment:
We should selectively wait only for the Ingress Controller, Dashboard UI and Kubeless UIs to be ready. That's the minimum needed to get to use the dashboard command, and the Dashboard UI has it's own wait loop for the API.

Reply:
I think that's fine to wait for all to be ready. If the kubeless-ui is ready but for some reasons the kubeless isn't, people might see it's weird because function can't be deployed from the ui.

Reply:
Well that's fine, but I don't think we should wait on the Monocular API...